content/rvm/install.md

```diff
@@ -4,16 +4,27 @@ title: Installing RVM
 
 # Installing RVM
 
-## Quick (guided) Install
+RVM supports most UNIX like systems and Windows (with Cygwin). The basic requirements are `bash`, `curl`, `gpg2` and overall GNU version of tools - but RVM tries to autodetect it and install anything that is needed.
 
-Before any other step install [mpapis](/authors/mpapis/) [public key](https://keybase.io/mpapis) (might need `gpg2`) (see [security](/rvm/security/))
+## Install GPG keys
+
+As a first step install [mpapis](/authors/mpapis/) [public key](https://keybase.io/mpapis) used to verify installation package to ensure [security](/rvm/security/).
 
     gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
     
-**Note: hkp://keys.gnupg.net can sometimes timeout you may want to use different keyserver
-- hkp://pgp.mit.edu
+***Note:*** *hkp://keys.gnupg.net can sometimes timeout you may want to use different keyserver, for example:*
```

Comment:
this note is still valid for `gpg` but if we fix the line above to use `gpg2` - a lot less users will hit this problem